ci/tests/mgmtcenter/mgmt_center_test.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2015 iNuron NV
+#
+# Licensed under the Open vStorage Non-Commercial License, Version 1.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.openvstorage.org/OVS_NON_COMMERCIAL
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from ovs.dal.lists.pmachinelist import PMachineList
+from ovs.dal.lists.mgmtcenterlist import MgmtCenterList
+from ovs.dal.hybrids.mgmtcenter import MgmtCenter
+from ovs.lib.mgmtcenter import MgmtCenterController
+from ci.tests.general import general
+from ci import autotests
+from nose.plugins.skip import SkipTest
+
+testsToRun = general.get_tests_to_run(autotests.get_test_level())
+
+
+def setup():
+    management_centers = MgmtCenterList.get_mgmtcenters()
+    if len(management_centers) >= 1:
+        mgmtcenter = management_centers[0]
+    else:
+        mgmtcenter = MgmtCenter()
+        mgmtcenter.name = general.test_config.get("mgmtcenter", "name")
+        mgmtcenter.username = general.test_config.get("mgmtcenter", "username")
+        mgmtcenter.password = general.test_config.get("mgmtcenter", "password")
+        mgmtcenter.ip = general.test_config.get('main', 'grid_ip')
+        mgmtcenter.type = 'OPENSTACK'
+        mgmtcenter.port = 443
+    mgmtcenter.save()
```

Comment:
we should use the api to create this ... not directly in the model, we used to do this in the past via jenkins